lib/scsi_defs.h

```diff
@@ -195,6 +195,100 @@ enum class PageCode : uint8_t {
   kLogicalBlockProvisioningVpd = 0xb2
 };
 
+// SCSI Reference Manual Table 461
+// https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+enum class Association : uint8_t {
```

Comment:
great structs but seems extra in the future tbh unless we have code reuse. seems to me that inquiry structs returned are one-off

Reply:
I agree that it could be one off but it increases the readability of the code quite a lot. I made this change after @oicirtap suggested that I should make variables for all the struct field initializers. Although, other people might feel different.

CC: @eyhhuang @eyhhuang @Ralith @toadchild @w-michal

Reply:
I'm generally in favor of symbolic constants over magic numbers in code, even if they aren't used repeatedly.